USING.md

```diff
@@ -0,0 +1,56 @@
+
+## What is OPAM
+
+OPAM is the package manager for the OCaml programming language, the language 
+in which Coq is implemented.
+Instructions on
+[how to install OPAM](http://opam.ocaml.org/doc/Install.html)
+itself are available on the OPAM website.
+
+By following the instructions on this web page one installs the last stable
+version of Coq and all additional packages in the directory `~/opam-coq.@COQV@`.
+Instructions target an OPAM newcomer.
+
+## Using OPAM to install Coq
+
+Once the `opam` command is available, i.e. OPAM is installed, one can
+proceed as follows:
+
+    export OPAMROOT=~/opam-coq.@COQV@ OPAMJOBS=2
```

Comment:
Split it on two lines so that you can add comments like "target installation directory" and "number of processor cores"